### DISCUSSION TITLE
using command line arguments

### DISCUSSION THREAD

**Discussion by hiszd** [tmuxinator/tmuxinator#903] [Q&A]
Using command line arguments has been great so far and I intend to use them always, but I'm not sure how to do the thing I want to do now.
I want to run a command like ``tmuxinator start rdev user 127.0.0.1``
and have a config for rdev like this:
```
# .config/tmuxinator/rdev.yml

name: rdev-<%= @args[1] %>
root: ~/

# Controls whether the tmux session should be attached to automatically. Defaults to true.
attach: true

windows:
  - server: ssh <%= @args[0] %>@<%= @args[1] %>
```

But the arguments don't get put into the command, or the config just doesn't know what to do with them.
Is there a correct way of doing this?

**Comment by hschne** [comment]
@hiszd I tried with your exact config and it worked as expected for me. I'm getting a new tmux session with `ssh firstarg@secondarg
` :thinking: 

Do you still experience this issue, and if so, what Tmux version are you on?

**Reply by hiszd** [reply]
It has been a VERY long time since I opened this thread. I will try this again and see if this issue persists.

**Reply by hschne** [reply]
Better late than never! 🤡
